modules/kubernetes-addons/velero/values.yaml

```diff
@@ -0,0 +1,24 @@
+configuration:
+  provider: aws
+  backupStorageLocation:
+    bucket: ${s3-bucket-name}
+
+serviceAccount:
+  server:
+    create: false
+    name: ${velero-sa-name}
+    annotations:
+    labels:
+image:
+  repository: velero/velero
+  tag: v1.4.2
+  pullPolicy: IfNotPresent
+  imagePullSecrets: []
+  
+initContainers:
+   - name: velero-plugin-for-aws
+     image: velero/velero-plugin-for-aws:v1.2.0
+     imagePullPolicy: IfNotPresent
+     volumeMounts:
+       - mountPath: /target
+         name: plugins
```

Comment:
Can be removed is this is default configuration

Reply:
default configuration is different

Reply:
image:
  repository: velero/velero
  tag: v1.8.1
  # Digest value example: sha256:d238835e151cec91c6a811fe3a89a66d3231d9f64d09e5f3c49552672d271f38.
  # If used, it will take precedence over the image.tag.
  # digest:
  pullPolicy: IfNotPresent
  # One or more secrets to be used when pulling images
  imagePullSecrets: []
  # - registrySecretName